test/tests_for_integration/api_to_db_test.py

```diff
@@ -84,12 +104,65 @@ def auth_url(config, mongo_client):
     _clean_auth_db(mongo_client)
 
 
-# TODO start the ee2 service
-# TODO wipe the ee2 database between every test
+# side effect: modifies the config
```

Comment:
is this likely to be an issue? would it be safer to clone the config first?

Reply:
I added some comments to the method and method invocation to explain the rationale here